modules/imap/functions.php

```diff
@@ -295,10 +295,6 @@ function format_imap_message_list($msg_list, $output_module, $parent_list=false,
             $icon = false;
         }
 
-        //if (in_array('draft', $flags)) {
-        //    $url = '?page=compose&list_path='.sprintf('imap_%s_%s', $msg['server_id'], $msg['folder']).'&uid='.$msg['uid'].'&imap_draft=1';
-        //}
```

Comment:
This seems some debugging code. @christer77 can you please review the diff thoroughly again, make sure you commit only what you want to commit and let me know to do a final review here?

Reply:
okay